lib/chatty/connection.ex

```diff
@@ -106,6 +107,30 @@ defmodule Chatty.Connection do
     {:noreply, %{updated_state | last_message_time: current_time()}}
   end
 
+  def handle_info({:ssl, sock, raw_msg}, %{sock: sock, user_info: user_info} = state) do
```

Comment:
This code duplication is bad. Make a local function and call it here and in `handle_info({:tcp, sock, raw_msg}, ...`.